runtime/src/token.rs

```diff
@@ -15,7 +15,7 @@ use system::{self, ensure_signed};
 #[cfg_attr(feature = "std", derive(Debug))]
 pub struct Token {
     pub id: TokenId,
-    pub decimals: u8,
+    pub decimals: u16,
```

Comment:
Not sure what for. Imho not much people anyone will want to have more than 255 decimals.